src/components/navigation/BottomTabNavigation.tsx

```
@@ -0,0 +1,231 @@
+      setLastScrollY(currentScrollY)
+    }
+
+    window.addEventListener('scroll', handleScroll, { passive: true })
+    return () => window.removeEventListener('scroll', handleScroll)
+  }, [lastScrollY])
+
+  // Set active tab based on pathname
+  useEffect(() => {
+    // Remove locale prefix from pathname for comparison
+    const pathSegments = pathname.split('/')
+    const pathnameWithoutLocale = '/' + pathSegments.slice(2).join('/') || '/'
```

ESLint

[error] Unexpected string concatenation.

(prefer-template)